CONTRIBUTING.md

```diff
@@ -242,13 +242,22 @@ but the consistency is worth it.
     class Foo[F[_], A, B] // good
     class Foo[A, F[_], B] // bad
 
-Define type lambdas with either lower case or greek letters as the type parameters. This helps to distinguish them
-from the applied types.
+Define type lambdas using the kind-projector plugin. Raw type-lambdas should be defined with either lower case or greek
+letters as the type parameters. This helps to distinguish them from the applied types. Type Lambdas should use the 
+kind-projector syntax. Here's some inline syntax examples:
 
-    ({type λ[α]=F[X, α]})#λ
-    ({type λ[α, β]=F[X, α, β]})#λ
-    ({type l[a]=F[X, a]})#l
-    ({type l[a, b]=F[X, a, b]})#l
+    F[X, ?]     ===  ({type λ[α] = F[X, α]})#λ
+    F[X, ?, ?]  ===  ({type λ[α, β] = F[X, α, β]})#λ
+    F[X, ?]     ===  ({type l[a] = F[X, a]})#l
+    F[X, ?, ?]  ===  ({type l[a, b] = F[X, a, b]})#l
+
+If using Lambda Type Function syntax (ie. when the kind-projector's inline syntax is insufficent), use greek letters for
+parameters.
+
+    λ[α => (α, α)]  ===  ({type λ[α] = (α, α)})#λ
+    λ[α[_] => F[α]  ===  ({type λ[α[_]] = F[α]})#λ
```

Comment:
@runarorama Not sure what you mean here, markdown is just four-space indent. I'm merely following the existing style…